go/vt/wrangler/switcher_dry_run.go

```diff
@@ -24,7 +24,7 @@ import (
 	"strings"
 	"time"
 
-	"vitess.io/vitess/go/maps2"
+	"golang.org/x/exp/maps"
```

Comment:
This should have a  newline after this one before the `vitess.io` imports.

Reply:
Aight!